packages/react-native-win32/src/Libraries/Components/Button/ButtonWin32.tsx

```diff
@@ -26,11 +26,11 @@ export class ButtonWin32 extends React.Component<IButtonWin32Props, IButtonWin32
 
   public render() {
     const viewProps: IViewWin32Props = {
-      acceptsKeyboardFocus: true,
       accessible: true,
       accessibilityLabel: this.props.accessibilityLabel || this.props.title,
       accessibilityRole: 'button',
       accessibilityState: this.state.accessibilityState,
+      focusable: true,
```

Comment:
FYI @falselobster we will need to update NetUI view managers for this before 0.63. Should be a trivial change though. Is there a path I should file this under?